### PR TITLE
Fix some typos in icon_theme.dart

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: packages/flutter/lib/src/material/icon_theme_data.dart [hunk 1]
```diff
@@ -11,7 +11,7 @@ import 'dart:ui' as ui show lerpDouble;
 /// widget subtree.
 ///
 /// To obtain the current icon theme, use [IconTheme.of]. To convert an icon
-/// theme to a version with all the fields filled in, use [fallback].
+/// theme to a version with all the fields filled in, use [new fallback].
```
Comment (abarth) [inline packages/flutter/lib/src/material/icon_theme_data.dart]: Does that linkify correct?

**Comment by abarth** [comment]
LGTM

File: packages/flutter/lib/src/material/icon_theme_data.dart (hunk 1 above)
Comment (Hixie) [inline packages/flutter/lib/src/material/icon_theme_data.dart]: Yeah that's how you link to constructors.

**Comment by HansMuller** [comment]
Currently '[new fallback]' isn't rendered as a link: https://docs.flutter.io/flutter/material/IconThemeData-class.html

**Comment by Hixie** [comment]
`[fallback]` wasn't either... maybe it has to be `[new IconThemeData.fallback]` ?

**Comment by Hixie** [comment]
->https://github.com/flutter/flutter/issues/9778

**Pull request merged**